Northstar.Custom/mod/scripts/vscripts/sh_message_utils.gnut

```diff
@@ -453,6 +454,7 @@ void function AnnouncementMessageHandler_Threaded()
 		Announcement_SetPriority( announcement, client.announcementQueue[0].priority )
 		Announcement_SetSoundAlias( announcement, SFX_HUD_ANNOUNCE_QUICK )
 		Announcement_SetStyle( announcement, client.announcementQueue[0].style )
+		Announcement_SetIcon( announcement, StringToAsset( strip( client.announcementQueue[0].image ) ) )
```

Comment:
Seems weird that they are stored in the struct as a string and converted to asset here but eh, probably still the best way to do it